src/PrestaShopBundle/Resources/config/services/adapter/shop_feature.yml

```diff
@@ -0,0 +1,25 @@
+services:
+  _defaults:
+    public: true
+
+  prestashop.adapter.combination_feature:
```

Comment:
```suggestion
  prestashop.adapter.combination.feature:
```
?

Reply:
this is not a new service so it would break BC. I've just moved these services from `src/PrestaShopBundle/Resources/config/services/adapter/feature.yml` to `src/PrestaShopBundle/Resources/config/services/adapter/shop_feature.yml` because I needed `feature.yml` name :slightly_smiling_face:

Reply:
Can you create an alias, we add this discussion on Gitter few days ago with @sarjon.
It's even better to have:
`prestashop.adapter.feature.SOMETHING`
`Namespace.Context.Type.Name`
wdyt @matks ?

Reply:
Yes, we need to avoid as much as possible BC 😞we can only have some if there is a really good reason behind. Here, keeping the alias is not an big maintainance burden so we can keep it.

Reply:
but I didn't change the service name in this PR at all (I just changed the yaml file name), do we really need to introduce a new alias?

Reply:
Indeed this is another issue that will be solved in another PR.
The issue is that name "prestashop.adapter.combination_feature" is wrong and we should fix the name *and* keep the previous name available as an alias to avoid BC